La-Mula-App-main/src/main/kotlin/com/lamulaapp/controller/dto/UsersDto.kt

```diff
@@ -0,0 +1,18 @@
+package com.lamulaapp.controller.dto
+
+import com.lamulaapp.domain.Roles
+import java.time.LocalDateTime
+import java.util.UUID
+
+data class UsersDto(
+    val id: UUID? = null,
+    val idRole: Roles? = null,

```

Comment:
Usar RoleDto en el tipo de dato y nombrarlo correctamente.

```suggestion
    val roleDto: RoleDto? = null,

```